include/service.h

```diff
@@ -45,7 +45,7 @@ class Service {
                             std::vector<discovered_alleles>& ans);
 
     /// Genotype a set of samples at the given sites, producing a BCF file.
-    Status genotype_sites(const genotyper_config& cfg, const std::string& sampleset, const std::vector<unified_site>& sites, const std::string& filename, consolidated_loss& dlosses);
+    Status genotype_sites(const genotyper_config& cfg, const std::string& sampleset, const std::vector<unified_site>& sites, const std::string& filename, consolidated_loss& dlosses, bool write_vcf=false);
```

Comment:
Adding an option to write to uncompressed vcf for easy `diff` without needing to call `bcftools view`